ruby-challenges.rb

```diff
@@ -10,23 +10,79 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# pseudocode: taking in an array of words create a return array called same_letter_array that takes the given array of strings and uses the built in method .select(which from researching requires a block and requires something that evaluates to true or false, in thise case if the string includes the letter we are looking for)then use the built in method .includes? pass the string of the given letter to find, make a p statement returning your created array, and it will return your strings with the same letter.
+
+
+beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+
+same_letter_array = beverages_array.select { |str| str.include?(letter_o) } # or letter_t for other example.
```

Comment:
You have the basic idea but you want to create a method that you can pass in an array and a single letter.